docs/modules/trino/examples/usage-guide/trino-insecure.yaml

```diff
@@ -31,18 +31,28 @@ spec:
       default:
         replicas: 1
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hive-credentials
+type: Opaque
+stringData:
+  username: APP
+  password: mine
+---
 apiVersion: hive.stackable.tech/v1alpha1
 kind: HiveCluster
 metadata:
   name: simple-hive-derby
 spec:
   image:
-    productVersion: 4.0.0
+    productVersion: 4.1.0
+    custom: oci.stackable.tech/sdp/hive:4.1.0-stackable0.0.0-dev-amd64
```

Comment:
what is different about this image? why didn't we need it in 4.0.0? (maybe add a brief explanation)